package.json

```diff
@@ -34,7 +34,7 @@
     "lodash": "^4.17.20",
     "pdfjs-dist": "2.0.943",
     "regenerator-runtime": "^0.13.7",
-    "sbc-common-components": "2.4.39",
+    "sbc-common-components": "2.4.41",
```

Comment:
For updated Base Address (with classnames on fields and whitespace above delivery instructions).